src/main/java/com/greenfoxacademy/opal/kalendaryo/kalendaryo/controllers/NotificationController.java

```diff
@@ -16,13 +21,16 @@
     EventResponseService eventResponseService;
 
     @PostMapping(value = "/noti")
```

Comment:
@nezihcihan can we have an actual valueable name for this endpoint? pretty pls :D